spec/features/images/01_band_can_upload_image_spec.rb

```diff
@@ -1,7 +1,16 @@
 require "rails_helper"
 
 feature "band photo" do
+  let!(:test_user) {User.create!(email: "test_user@mail.com", password: "password", username: "test_user_2", first_name: "test", role: "admin")}
   scenario "user uploads a band profile photo" do
+    visit "/"
+    click_link "Sign In"
+
+    fill_in 'Email', with: test_user.email
+    fill_in 'Password', with: test_user.password
+
+    click_button "Log in"
```

Comment:
ill allow this, but you can skip all of these many lines with the `sign_in` helper.